packages/terra-form-select/src/shared/_FrameUtil.js

```diff
@@ -25,19 +25,22 @@ class FrameUtil {
    * @param {ReactNode} target - The select wrapper.
    * @param {ReactNode} dropdown - The dropdown.
    * @param {number} maxHeight - The maxHeight of the dropdown dropdown.
+   * @param {Object} currentState - The current component state.
    * @param {boolean} useSemanticDropdown - If the dropdown should be rendered semantically instead of in a portal
    * @return {Object} - The calculated dropdown attributes.
    */
-  static dropdownPosition(props, target, dropdown, maxHeight, useSemanticDropdown = false) {
+  static dropdownPosition(props, target, dropdown, maxHeight, currentState, useSemanticDropdown = false) {
     const style = ({ ...props }).style || {};
     const { height } = dropdown.getBoundingClientRect();
     const { bottom, width, top } = target.getBoundingClientRect();
 
     const spaceBelow = window.innerHeight - bottom;
     const maximumHeight = parseInt(style.maxHeight || maxHeight, 10) || Infinity;
-    const canFitBelow = maximumHeight < spaceBelow || height < spaceBelow || spaceBelow > top;
+    let canFitBelow = (maximumHeight < spaceBelow || height < spaceBelow || spaceBelow > top);
+    if (useSemanticDropdown && currentState !== undefined && currentState.resizeOnSearch && currentState.isAbove && canFitBelow) {
+      canFitBelow = false;
```

Comment:
why are we not allowing the dropdown display below here ? will this logic not impact both the DOM and Portal rendering ? should we check the isTouchAccessible prop here ?

Reply:
useSemanticDropdown is the isTouchAccessible prop

Reply:
without this condition dropdown displays below the form-select on search (when it has only one option in list) which seems to be expected behaviour of form-select when there is only one option to disaply but dropdown will not resize once it is displayed below the form-select as shown in below image. even after clearing the search value size of the dropdown remains unchanged. ( I am not sure why it is not resizing to it's default height on clearing the search value )


![form-select-detach](https://user-images.githubusercontent.com/12869809/78809567-c3a38780-79e4-11ea-9931-36a0d2f2ff62.gif)

Reply:
Resolved a2c35f5